File: server/Models/Recipe.cs
namespace all_spice.Models;

public class Recipe : DbItem<int>
{
    public string Title { get; set; }
    public string Img { get; set; }
    public string Category { get; set; }
    public string CreatorId { get; set; }
    public Profile Creator { get; set; }
    public int FavoriteCount { get; set; }
#nullable enable
    public string? Instructions { get; set; }
}

File: client/src/models/Recipe.js
export class Recipe {
	constructor(data) {
		this.id = data.id;
		this.createdAt = new Date(data.createdAt);
		this.updatedAt = new Date(data.updatedAt);
		this.creatorId = data.creatorId;
		this.creator = data.creator;
		this.category = data.category;
		this.favoriteCount = data.favoriteCount;
		this.imgUrl = data.img;
		this.instructions = data.instructions;
		this.title = data.title;
		this.favoriteId = data.favoriteId ? data.favoriteId : null;
	}
}
